### PR TITLE
fix: configure sandbox permissions for Chrome on Windows

### LINKED ISSUE #12471: Read this if you experience PDF printing hanging on Windows
With M125 Chrome uses sandboxes for printing (https://crbug.com/338553158) which may require additional permissions on the Chrome installation folder:

See https://pptr.dev/troubleshooting#chrome-reports-sandbox-errors-on-windows for how to set permissions.

### PR DESCRIPTION
**Pull request by OrKoN**
Configure permissions by invoking setup.exe.

Closes https://github.com/puppeteer/puppeteer/issues/12471

### CODE REVIEW

File: packages/browsers/src/fileUtil.ts [hunk 1]
```diff
@@ -89,3 +89,21 @@ async function installDMG(dmgPath: string, folderPath: string): Promise<void> {
     await exec(`hdiutil detach "${mountPath}" -quiet`);
   }
 }
+
+export async function configureWindowsSandboxPermissionsForChrome(
```
Comment (Lightning00Blade) [inline packages/browsers/src/fileUtil.ts]: This seems out of place in this file, it does not have relation to files.

Comment (OrKoN) [reply packages/browsers/src/fileUtil.ts]: it sets file permissions

**Review by Lightning00Blade — APPROVED** [review]

**Comment by OrKoN** [comment]
@Lightning00Blade FYI I have updated this to use setup.exe but it should start working only with M127

**Review by Lightning00Blade — APPROVED** [review]

**Review by Lightning00Blade — APPROVED** [review]

**Comment by kblok** [comment]
@OrKoN does chromium have the same issue?
I have people reporting that PDF functions hang on Chromium.

**Comment by OrKoN** [comment]
I believe it might but we are not testing with Chromium anymore. Does the same workaround help?

**Comment by kblok** [comment]
@OrKoN, the chromium zip file doesn't have a `setup.exe`, so we won't be able to do this.

Shouldn't we communicate this whether as a validation or in some doc?

**Comment by OrKoN** [comment]
@kblok the workaround without setup.exe is documented here https://pptr.dev/troubleshooting#chrome-reports-sandbox-errors-on-windows

**Comment by OrKoN** [comment]
we might also file a request to include setup.exe into Chromium, let me look into that

**Comment by kblok** [comment]
> @kblok the workaround without setup.exe is documented here https://pptr.dev/troubleshooting#chrome-reports-sandbox-errors-on-windows

Ah yeah. I haven't tried that.

**Comment by OrKoN** [comment]
let's see if the reviewers have any comments https://chromium-review.googlesource.com/c/chromium/src/+/5764985

**Pull request merged**